docs/content/references/cli/client.mdx

```diff
@@ -188,6 +188,211 @@ $ sui client dynamic-field 0x5
 ╰─────────────┴───────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
+### Send SUI or objects
+
+In this example, let's see how to transfer SUI or transfer an object from one address to another. First all, there two main commands for sending SUI or transfering objects: Pay and Transfer. 
+Each of these commands, has some sister commands: PaySui, PayAllSui, and TransferSui. The main difference between Pay and Transfer is that Pay allows you to send coin to multiple recipients, whereas Transfer only accepts one recipient. Another important difference is that Pay and Transfer require a gas object that is different than the one used to send the coins, which is used to pay gas fees. 
```

Comment:
Ditto on formatting of command names -- let's use the form that's recognisable from the CLI (e.g. `pay-sui`, `pay-all-sui` etc). Also, I would describe the differences between these commands as follows:

- Commands that end in `-sui` deal with Sui's native coin, and they use the input coins to pay for gas and for transferring.
- `pay-` commands typically deal with coins and handle smashing for you, whereas `transfer-` commands can handle the transfer of any object that has public transfer (has the `store` ability).

Some special cases:
- `pay-all-sui` is a special case of `pay-sui` that offers a way to transfer the entire balance after smashing.
- `transfer-sui` is a legacy command -- it has been entirely superseded by `pay-sui` or `pay-all-sui` depending on whether an amount is specified or not.

Reply:
Thanks, updated!